docs/learning/backend/Channel.md

```diff
@@ -1,44 +1,44 @@
-# Working With Channels in iModels
+# Editing Channels in iModels
```

Comment:
i think it should be `Working with "Editing Channels" in iModels` with the quotes. Otherwise "Editing" looks like a verb.